api.py

```diff
@@ -367,7 +388,134 @@ def set_generic_data(dataset_id, key, data_string, **kwargs):
     return ingest_entities(dataset_id, {'entities': [entity]}, **kwargs)
 
 
+def convert_coordinates(point):
+    if len(point) != 2:
+        raise KeyError('A point should only have two dimensions', point)
+    if 'lat' and 'lon' in point:
+        coord = {
+            'x': float(point['lon']),
+            'y': float(point['lat'])
+        }
+    elif 'x' and 'y' in sample['point']:
+        coord = {
+            'x': float(point['x']),
+            'y': float(point['y'])
+        }
+    else:
+        raise KeyError('invalid coordinate', point)
+
+    coord['z'] = 0
+
+    return coord
+
+
+def convert_samples_to_entity_set(sample_list):
+    conduce_keys = ['id', 'kind', 'time', 'point', 'path', 'polygon']
+    entities = []
+    for sample in sample_list:
+        if not isinstance(sample['time'], datetime):
+            raise TypeError('time must be a datetime object', sample['time'])
+        coordinates = []
+        if 'point' in sample:
+            coordinates = [convert_coordinates(sample['point'])]
+        if 'coordinates' in sample:
```

Comment:
did you mean if 'path' in sample?

Reply:
No this new API method uses new data structures outlined in the new sphinx docs.  See PR #43.  I'll send you an invite to hosted sphinx documentation.

Reply:
Thanks I had some inconsistent naming.  Looks like I didn't get it all resolved.